src/mono/mono/mini/interp/interp.c

```diff
@@ -8945,4 +8945,21 @@ mono_jiterp_get_simd_opcode (int arity, int index)
 #endif
 }
 
+EMSCRIPTEN_KEEPALIVE void
+mono_jiterp_get_opcode_info (int opcode, void *_result)
+{
+	struct {
+		const char *name;
+		int length_u16, num_dregs, num_sregs, opargtype;
+	} *result = _result;
+	g_assert (result);
+	g_assert (opcode >= 0);
```

Comment:
Nit: Could be easier to read if `g_assert (opcode >= 0 && opcode <= MINT_LASTOP);`.